test/features/binary_build.feature

```diff
@@ -0,0 +1,18 @@
+@binary
+Feature: Use binary build to deploy the service
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+
+  @springboot
+  Scenario: Deploy jbpm-springboot-example using binary build
+    Given Clone Kogito examples into temporary directory
```

Comment:
would use `local` instead of temporary as we have then step `Local example service` wdyt ?